nurseconnect/templates/core/section_page.html

```diff
@@ -7,62 +6,87 @@
+                {#              {% if featured_articles %}#}
+                {#              {% for article in featured_articles %}#}
+                {#                <h3><a href="{% pageurl article %}">{{article.title}}</a></h3>#}
+                {#                <p>{{article.subtitle}}</p>#}
+                {#              {% endfor %}#}
+                {#              {% endif %}#}
+                {#            </li>#}
+                {#          </ul>#}
             </li>
-          </ul>
-        </h2>
-      {% endfor %}
+        {% endfor %}
+    </ul>
+    <ul class="Breadcrumb-back">
+        <li class="Breadcrumb-item">
+            <a class="Breadcrumb-link">
+                <span class="Breadcrumb-backIcon"><</span>
+            </a>
+        </li>
 
-      {% load_child_articles_for_section self as articles %}
-      {% for article in articles %}
-      <ul class="list-block">
-        <li>
-          {% if article.image %}
-          <a href="{% pageurl article %}">{% image article.image width-60 %}</a>
-          {% endif %}
-          <h3><a href="{% pageurl article %}">{{article.title}}</a></h3>
-          <p>{{article.subtitle}}</p>
+            <li class="Breadcrumb-item Breadcrumb-item--active"><a class="Breadcrumb-link" href={{ self.get_parent.url }}>Back</a>
+{#                {% breadcrumbs %}#}
         </li>
-      </ul>
-      {% endfor %}
+    </ul>
 
-      {% if articles %}
-      <div class="step-links l-indent l-indent--tall">
-        {% if articles_paginated.has_previous %}
-            <a href="?p={{ articles_paginated.previous_page_number }}">&larr;</a>
-        {% endif %}
-        <span class="current">
-            Page {{ articles_paginated.number }} of {{ articles_paginated.paginator.num_pages }}
-        </span>
-        {% if articles_paginated.has_next %}
-            <a href="?p={{ articles_paginated.next_page_number }}">&rarr;</a>
-        {% endif %}
-      </div>
-      {% endif %}
-      
-    </div>
+    {% load_child_articles_for_section self as articles %}
+    {% for article in articles %}
+        <ul class="ListView">
+            <li class="ListItemWrapper">
+                <a class="ListItem ListItem--withImage" href="{% pageurl article %}">
+                    <h3 class="ListItem-title">{{ article.title }}</h3>
+                    <h4 class="ListItem-subtitle">{{ article.subtitle }}</h4>
+                    <p class="ListItem-desc">{{ article.description }}</p>
+                    {% if article.image %}
+                        <a class="ListItem-image" href="{% pageurl article %}">{% image article.image width-60 %}</a>
+                    {% endif %}
+                </a>
+            </li>
+        </ul>
+    {% endfor %}
+
+    {% if articles %}
+        <ul class="Breadcrumb-back">
+            <li class="Breadcrumb-item">
+                <a class="Breadcrumb-link">
+                    <span class="Breadcrumb-backIcon"><</span>
```

Comment:
This will most likely need to be removed since the designs have changed. Not sure if there will ever be a back button on the site.